unstructured/__version__.py

```diff
@@ -1 +1 @@
-__version__ = "0.12.2-dev0"  # pragma: no cover
+__version__ = "0.12.3-dev1"  # pragma: no cover
```

Comment:
shouldn't be bumping both subminor AND dev version, right?

Reply:
ahh, nevermind, looking at the changelog history it looks like the last bump to 2-dev0 was incorrect...given that we have both `0.12.2-dev0` and `0.12.2` which should never happen. looks like we should have had a `0.12.3-dev0` and your PR is bumping us from that to dev1. got it.